test/e2e/framework/manifest/manifest.go

```diff
@@ -85,3 +91,48 @@ func StatefulSetFromManifest(fileName, ns string) (*appsv1.StatefulSet, error) {
 	}
 	return &ss, nil
 }
+
+// GetDaemonSetFromURL reads from a url and returns the daemonset in it.
+func GetDaemonSetFromURL(url string) (*appsv1.DaemonSet, error) {
```

Comment:
nit: can we rename it to `DaemonSetFromURL`.
For the most part, it does "Get" a daemonset from a URL but it also creates it in the last step.

Reply:
ok .  i will do it.

Reply:
Yeah, my point was `Get` part.
So it is nice to rename it to `DaemonSetFromURL`.